src/Router.sol

```diff
@@ -50,11 +50,13 @@ contract Router is IRouter, EIP712, FeeVerifier {
         _;
     }
 
-    constructor(address wrappedNative, address pauser_, address feeCollector_) EIP712('Composable Router', '1') {
+    constructor(address wrappedNative, address owner_, address pauser_, address feeCollector_) EIP712('Composable Router', '1') {
         currentUser = _INIT_USER;
         agentImplementation = address(new AgentImplementation(wrappedNative));
+        transferOwnership(owner_);
         _setPauser(pauser_);
         _setFeeCollector(feeCollector_);
+
     }
```

Comment:
Seems it's not formatted.
`./node_modules/.bin/prettier  --config .prettierrc --write .`


![image](https://github.com/dinngo/composable-router-contract/assets/4598616/29a7766b-af4d-4802-9621-f4de92c94e93)